integration-tests/suites/procfs_scraper.go

```diff
@@ -0,0 +1,105 @@
+package suites
+
+import (
+	"os"
+	"strconv"
+	"time"
+
+	"github.com/stackrox/collector/integration-tests/suites/common"
+	"github.com/stretchr/testify/assert"
+)
+
+type ProcfsScraperTestSuite struct {
+	IntegrationTestSuiteBase
+	ServerContainer             string
+	TurnOffScrape               bool
+	RoxProcessesListeningOnPort bool
+}
+
+// Launches nginx container
+// Launches gRPC server in insecure mode
+// Launches collector
+// Note it is important to launch the nginx container before collector, which is the opposite of
+// other tests. The purpose is that we want ProcfsScraper to see the nginx endpoint and we do not want
+// NetworkSignalHandler to see the nginx endpoint.
+func (s *ProcfsScraperTestSuite) SetupSuite() {
+
+	s.metrics = map[string]float64{}
+	s.executor = common.NewExecutor()
+	s.StartContainerStats()
+	s.collector = common.NewCollectorManager(s.executor, s.T().Name())
+
+	s.collector.Env["COLLECTOR_CONFIG"] = `{"logLevel":"debug","turnOffScrape":` + strconv.FormatBool(s.TurnOffScrape) + `,"scrapeInterval":2}`
+	s.collector.Env["ROX_PROCESSES_LISTENING_ON_PORT"] = strconv.FormatBool(s.RoxProcessesListeningOnPort)
+
+	s.launchNginx()
+
+	err := s.collector.Setup()
+	s.Require().NoError(err)
+
+	err = s.collector.Launch()
+	s.Require().NoError(err)
+	time.Sleep(10 * time.Second)
+
+	s.cleanupContainer([]string{"nginx"})
+	time.Sleep(10 * time.Second)
+
+	err = s.collector.TearDown()
+	s.Require().NoError(err)
+
+	s.db, err = s.collector.BoltDB()
+	s.Require().NoError(err)
+}
+
+func (s *ProcfsScraperTestSuite) launchNginx() {
+	image := "nginx:1.14-alpine"
+
+	err := s.executor.PullImage(image)
+	s.Require().NoError(err)
+
+	// invokes default nginx
+	containerID, err := s.launchContainer("nginx", image)
+	s.Require().NoError(err)
+	s.ServerContainer = common.ContainerShortID(containerID)
+
+	time.Sleep(10 * time.Second)
+}
```

Comment:
This can probably be turned into a more generic function, I just saw `process_network.go` also launches nginx containers.

Reply:
Agreed, will look to implement in a follow up to consolidate a lot of the generic functionality throughout the tests